src/Symfony/Component/HttpFoundation/File/UploadedFile.php

```diff
@@ -141,9 +141,26 @@ public function guessClientExtension()
      * It is extracted from the request from which the file has been uploaded.
      * Then it should not be considered as a safe value.
      *
-     * @return int|null The file size
+     * @deprecated since 4.1 will be removed in 5.0 use getFileSize instead.
+     *
+     * @return int|null The file sizes
      */
     public function getClientSize()
+    {
+        @trigger_error(sprintf('%s is deprecated since 4.1 and will be removed in 5.0 use %s::%s.', __METHOD__, __CLASS__, 'getSize'), E_USER_DEPRECATED);
+
+        return $this->size;
+    }
+
+    /**
+     * Returns the file size.
+     *
+     * It is calculated from $_FILES['fileupload']['size'] which is computed by PHP.
+     * Then it should be considered as a safe value.
+     *
+     * @return int|null The file size
+     */
+    public function getFileSize()
```

Comment:
`getClientSize` becomes `getSize`?

Reply:
which you are actually suggesting above :)

Reply:
It's not the same thing I guess, `getSize` already exists.

Reply:
Right, my bad. `\SplFileInfo::getSize — Gets file size`.

So what do _we_ suggest here? Another file size? Or should we override getSize to avoid re-calculating (or forward if $size is null).

`getUploadedSize` maybe?  Would it make sense?